### ISSUE TITLE
Module for querying URLhaus

### ISSUE THREAD

**Issue by abusech** [MISP/misp-modules#271]
It would be cool to have a module that enriches certain information with data from the URLhaus API:
https://urlhaus-api.abuse.ch/

I'm specially thinking about querying the URLhaus API for a malware sample (MD5 hash or SHA256 hash) or a URL:
https://urlhaus-api.abuse.ch/#urlinfo
https://urlhaus-api.abuse.ch/#payloadinfo

**Comment by chrisr3d** [comment]
Sure, I will work on it soon and poke you once it is done within the next few days ;-)

**Comment by abusech** [comment]
Did you had any chance to look into this feature request yet?

**Comment by chrisr3d** [comment]
The [module](https://github.com/MISP/misp-modules/blob/master/misp_modules/modules/expansion/urlhaus.py) is available since the feature branch we created with other modules presenting the same new features (being able to return objects as well as attributes from modules) has been merged into the master branch.

It required some work on MISP core as well, which took me indeed some time, but it is merged as well, and available too. It will be released (new tagged version and so on) today hopefully, or during the week end.

By the way any comment, suggestion, idea to improve the module is very welcome
